Mathlib/RingTheory/Localization/NumDen.lean

```diff
@@ -105,9 +119,31 @@ theorem isInteger_of_isUnit_den {x : K} (h : IsUnit (den A x : A)) : IsInteger A
   rw [← mul_assoc, mul_inv_cancel d_ne_zero, one_mul, mk'_spec']
 #align is_fraction_ring.is_integer_of_is_unit_denom IsFractionRing.isInteger_of_isUnit_den
 
-theorem isUnit_den_of_num_eq_zero {x : K} (h : num A x = 0) : IsUnit (den A x : A) :=
-  num_den_reduced A x (h.symm ▸ dvd_zero _) dvd_rfl
-#align is_fraction_ring.is_unit_denom_of_num_eq_zero IsFractionRing.isUnit_den_of_num_eq_zero
+theorem isUnit_den_zero : IsUnit (den A (0 : K) : A) :=
+  num_den_reduced A 0 (by simp) dvd_rfl
+#align is_fraction_ring.is_unit_denom_of_num_eq_zero IsFractionRing.isUnit_den_zero
+
+variable (A)
```

Comment:
As I understand it, we're trying to avoid explicit binders in `variable`s. Can you move the `(A)` to the two declarations which need it?

Reply:
I'm not sure they actually need to have it explicit, so I just remove this